deployment/aws-batch/job-definitions/pfb-analysis-run-job.json

```diff
@@ -4,15 +4,36 @@
     "parameters": {},
     "containerProperties": {
         "image": "{image}",
-        "vcpus": 4,
-        "memory": 4096,
+        "vcpus": 2,
+        "memory": 12288,
         "command": [
             "/pfb/scripts/entrypoint.sh"
         ],
         "jobRoleArn": "",
-        "volumes": [],
-        "environment": [],
-        "mountPoints": [],
+        "volumes": [{
+            "name": "pgdata",
+            "host": {
+                "sourcePath": "/media/nvme0n1/pgdata"
```

Comment:
See other comment about how nvme SSDs get mounted in AWS.

`/dev/nvme0n1` is automatically exposed to the OS, then `/media/nvme0n1` is the mount point for `/dev/nvme0n1`, set via cloud-config. Docker automatically creates any subdirectories that are part of the volume mount point that don't exist.